mypyc/analysis/dataflow.py

```diff
@@ -248,6 +252,13 @@ def visit_assign(self, op: Assign) -> GenAndKill:
         else:
             return {op.dest}, set()
 
+    def visit_set_mem(self, op: SetMem) -> GenAndKill:
+        # Loading an error value may undefine the register.
+        if isinstance(op.src, LoadErrorValue) and op.src.undefines:
+            return set(), {op.dest}
```

Comment:
Since we don't modify `op.dest` but a value pointed to by it, this case can be removed.